WCFDataService/Client/System/Data/Services/Client/ClientEdmModel.cs

```diff
@@ -443,7 +447,8 @@ private EdmTypeCacheValue GetOrCreateEdmTypeInternal(IEdmStructuredType edmBaseT
                             List<IEdmProperty> loadedProperties = new List<IEdmProperty>();
                             foreach (PropertyInfo property in ClientTypeUtil.GetPropertiesOnType(type, /*declaredOnly*/edmBaseType != null).OrderBy(p => p.Name))
                             {
-                                IEdmProperty edmProperty = this.CreateEdmProperty(complexType, property);
+                                FieldInfo backingField = FindBackingField(property, this.ProxyBackingFieldNamingConvention);
```

Comment:
Ah, now I get it. The line you are referring to is related to properties ON a Complex type, not properties OF a complex type. As far as I remember you do not support navigation properties (and thus not DSC<> or ObservableCollection<> typed properties) on complex types. If that is correct, then yes, we could omit this. Whats your take on this?